packages/react-components/react-virtualizer/stories/VirtualizerScrollView/Default.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { VirtualizerScrollView } from '@fluentui/react-components/unstable';
+import { makeStyles } from '@fluentui/react-components';
+import { ThemeProvider } from '@fluentui/react';
+
+const useStyles = makeStyles({
+  root: {
+    maxHeight: '100vh',
+  },
+  child: {
+    height: '100px',
+    lineHeight: '100px',
+    width: '100%',
+  },
+});
+
+export const Default = () => {
+  const styles = useStyles();
+  const childLength = 1000;
+
+  return (
+    <ThemeProvider className={styles.root} applyTo="body">
+      <VirtualizerScrollView numItems={childLength} itemSize={100}>
```

Comment:
Something is wrong with this example? I see all the items rendered:

![image](https://user-images.githubusercontent.com/20744592/225108308-72d22178-a520-4f5e-81d1-e8827b9b97f6.png)

Reply:
My mistake, we were setting the height to be 100%, which expanded it to use the document body scrollview, we've added a limited height on the scroll container now (100vh) to ensure the scrollView itself is shown.